.circleci/config.yml

```diff
@@ -126,28 +146,42 @@ workflows:
               ignore:
                 - staging
                 - release
+      - validate_staging_schema:
+          filters:
+            branches:
+              only: master
+      - validate_production_schema:
+          filters:
+            branches:
+              only: release
       - push_staging_image:
           filters:
             branches:
               only: master
           requires:
             - test
             - acceptance
+            - validate_staging_schema
       - push_production_image:
           filters:
             branches:
               only: release
+          requires:
+            - validate_production_schema
       - publish_staging_assets:
           filters:
             branches:
               only: master
           requires:
             - test
             - acceptance
+            - validate_staging_schema
       - publish_production_assets:
           filters:
             branches:
               only: release
+          requires:
```

Comment:
WDYT about requiring the `validate_staging_schema` and `validate_production_schema` jobs in the `deploy_hokusai_staging` and `deploy_hokusai_production` jobs directly, instead of indirectly requiring them via dependent jobs (`publish_production_assets`, `push_production_assets`, etc.).

Reply:
I think the current implementation is better, since it avoids publishing the target image/assets when there's an incompatibility. (Or did I misunderstand?)

Reply:
Yes I did this so as to fail before publishing assets